Utils/layouts/banner.js

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { BannerSection, SubSection, BannerText } from "."
+import { i18n, withTranslation } from "../translation/i18n";
+import {Col, Heading, resolveMedia} from "@moneypensionservice/directories";
+import styled from "styled-components";
+
+import LogoEN from "./LogoEN.jsx"
+import LogoCY from "./LogoCY.jsx"
+
+export const MoneyHelperLogo = styled(Col)`
+  display: flex;
+  flex-direction: row;
+  ${resolveMedia.md`
+    justify-content: flex-end;
+ 
+`};
+  `;
+
+  export const LargeLogo = styled(MoneyHelperLogo)`
+  display: none;
+  ${resolveMedia.md`
```

Comment:
I think we would need to render the large/inline logos at a smaller breakpoint to be consistent with the design, such as using `resolveMedia.sm`.

Reply:
I don't think I understand what you meant here. We would probably pick this up tomorrow.

Reply:
Fixed.